src/backgrounds/_base.scss

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file Backgrounds
+ * @module Backgrounds
+ * @overview Background and matching text styles for page modules
+ * @todo: remove _backdrops.scss since this replaces it?
+ * @todo: rename TrialWarranty to be generic, update to use Starry Background
+ */
+
+.background--gray {
+  background: color(light-background);
+}
+
+.background--primary {
+  background-color: color(primary);
+  color: color(background);
+}
+
+.background--starry {
+  background-color: color(dark-blue);
+  background-image: url('https://casper.imgix.net/backgrounds/big-stars.png');
+  background-size: cover;
+  color: color(background);
+
+  &.is-stars-loaded {
+    background-image: none;
+  }
+
+  .background-content {
+    position: relative;
+    z-index: 2;
+  }
+}
+
+.background-stars {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 200%;
+  background-size: 100% 100%;
+  background-size: cover;
+  will-change: transform;
+}
+
+.background-stars--big {
+  background-image: url('https://casper.imgix.net/backgrounds/big-stars-small@2x.png');
+
+  @include respond-below(376px, width, 2) {
+    background-image: url('https://casper.imgix.net/backgrounds/big-stars-small@2x.png');
+  }
+
+
+  @include respond-below(376px, width, 3) {
+    background-image: url('https://casper.imgix.net/backgrounds/big-stars-small@3x.png');
+  }
+
+
+  @include respond-between(375px, 1024px, width) {
+    background-image: url('https://casper.imgix.net/backgrounds/big-stars-med.png');
+  }
+
+
+  @include respond-between(375px, 1024px, width, 2) {
+    background-image: url('https://casper.imgix.net/backgrounds/big-stars-med@2x.png');
+  }
+
+
+  @include respond-between(375px, 1024px, width, 3) {
+    background-image: url('https://casper.imgix.net/backgrounds/big-stars-med@3x.png');
+  }
+
+
+  @include respond-at-and-above(1025px, width) {
+    background-image: url('https://casper.imgix.net/backgrounds/big-stars.png');
+  }
+}
+
+.background-stars--small {
+  background-image: url('https://casper.imgix.net/backgrounds/small-stars-small@2x.png');
+
+  @include respond-below(376px, width, 2) {
+    background-image: url('https://casper.imgix.net/backgrounds/small-stars-small@2x.png');
+  }
+
+
+  @include respond-between(375px, 1024px, width) {
+    background-image: url('https://casper.imgix.net/backgrounds/small-stars-med.png');
+  }
+
+
+  @include respond-between(375px, 1024px, width, 2) {
+    background-image: url('https://casper.imgix.net/backgrounds/small-stars-med@2x.png');
+  }
+
+
+  @include respond-between(375px, 1024px, width, 3) {
+    background-image: url('https://casper.imgix.net/backgrounds/small-stars-med@3x.png');
+  }
+
+
+  @include respond-at-and-above(1025px, width) {
+    background-image: url('https://casper.imgix.net/backgrounds/small-stars.png');
+  }
+}
+
+h1,
+h2,
+h3 {
+  .background--primary &,
+  .background--starry & {
+    color: color(background);
+  }
+}
```

Comment:
Hey, @ipangandoyon! What's the best way for us to note requests for assets? This request is actually a long standing one to provide the starry night background as an SVG. Our current implementation is not very performant and unwieldy from a code maintenance perspective. 

This ask isn't urgent. It would be great if it could happen as part of the month iteration as it can be seen as part of what's necessary to release this pattern. However, it isn't necessary as long as we can get it tee'd up.